src/extensions/extensionInterface.ts

```diff
@@ -293,7 +298,12 @@ type SetWaveformsModeRecordingSelectionAction = {
     waveformsMode: 'geom' | 'vertical'
 }
 
-export type RecordingSelectionAction = SetRecordingSelectionRecordingSelectionAction | SetSelectedElectrodeIdsRecordingSelectionAction | SetVisibleElectrodeIdsRecordingSelectionAction | SetCurrentTimepointRecordingSelectionAction | SetTimeRangeRecordingSelectionAction | SetAmpScaleFactorRecordingSelectionAction | ScaleAmpScaleFactorRecordingSelectionAction | SetCurrentTimepointVelocityRecordingSelectionAction | SetWaveformsModeRecordingSelectionAction
+type SetRecordingSelectionAction = {
```

Comment:
Action to set the entire recording selection (important for use with useBufferedDispatch hook)